integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/KafkaContainer.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.container;
+
+import com.google.common.base.Preconditions;
+import java.net.Socket;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.apache.logging.log4j.util.Strings;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.Network;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
+import org.testcontainers.shaded.com.google.common.collect.ImmutableSet;
+
+public class KafkaContainer extends BaseContainer {
+
+  public static final int DEFAULT_BROKER_PORT = 9092;
+  private static final String DEFAULT_KAFKA_IMAGE =
+      System.getenv("GRAVITINO_CI_KAFKA_DOCKER_IMAGE");
+  private static final String DEFAULT_HOST_NAME = "gravitino-ci-kafka";
+
+  protected KafkaContainer(
+      String image,
+      String hostName,
+      Set<Integer> ports,
+      Map<String, String> extraHosts,
+      Map<String, String> filesToMount,
+      Map<String, String> envVars,
+      Optional<Network> network) {
+    super(image, hostName, ports, extraHosts, filesToMount, envVars, network);
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
```

Comment:
Can you please add the logic to copy logs out of container like what @yuqi1129 did.

Reply:
added